hi-ml-histopathology/src/histopathology/configs/run_ids.py

```diff
@@ -1,6 +1,6 @@
 # TODO: remove innereye references
 innereye_ssl_checkpoint = "hsharma_panda_explore:hsharma_panda_explore_1638437076_357167ae"
-innereye_ssl_checkpoint_binary = "hsharma_panda_tiles_ssl:hsharma_panda_tiles_ssl_1639766433_161e03b9"
+innereye_ssl_checkpoint_binary = "PANDA_SimCLR:PANDA_SimCLR_1654023538_7a0f1d00"
```

Comment:
minor thing: Can you try if you can drop the "PANDA_SimCLR:" prefix?

Reply:
Dropped, it works without the prefix.